modules/thermal_hydraulics/src/actions/CoupledHeatTransferAction.C

```diff
@@ -78,6 +78,13 @@ CoupledHeatTransferAction::validParams()
 
   params.addRequiredParam<std::string>("multi_app", "The name of the multi-app.");
 
+  params.addRangeCheckedParam<std::vector<Real>>(
```

Comment:
Should you update the test file ( [(modules/thermal_hydraulics/test/tests/actions/coupled_heat_transfer_action/master.i)](https://mooseframework.inl.gov/docs/PRs/26516/site/source/actions/CoupledHeatTransferAction.html#69562ab3-ad1d-4b8b-9133-56ee941f8ce1)  to remove the `bounding_box_padding` in the MultiApp block? My understanding is that it is not used with the GeneralField transfers.